pypsa/linopf.py

```diff
@@ -739,6 +739,13 @@ def define_growth_limit(n, sns, c, attr):
     ext_i = get_extendable_i(n, c)
     if "carrier" not in n.df(c) or n.df(c).empty:
         return
+
+    if n.carriers.max_relative_growth.notnull().any():
+        logger.warning(
+            "Max relative growth is not implemented for the native pypsa optimization framework. "
+            "Use the linopy framework with `n.optimize` instead."
+        )
```

Comment:
Sorry, seeing this message only now!

@FabianHofmann 

This will always trigger, since the default value for `max_relative_growth = 0`.

This should use
```
if (n.carrier.max_relative_growth > 0).any():
    ...
```

Reply:
Ah thanks, will fix it